docs/application/dotnet/guides/nui/text.md

```diff
@@ -402,6 +402,25 @@ field.HorizontalAlignment = HorizontalAlignment.Begin;
 ```
 
 <a name="textField4"></a>
+### Use Input Properties in TextField
+
+To change the text settings for new input text, you can use the Input properties, such as `InputColor` and `InputPointSize` of the [Tizen.NUI.BaseComponents.TextField](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextField.html) class.
+
+Especially, these properties should be used with some specific conditions.
+
+For instance, you can use these properties when the `TextChanged` event occurs or when the `Clicked` event of Button occurs.

```

Comment:
Are these the only two conditions for all Input properties?

Reply:
No, there are many conditions that app developers can set for Input properties.
The reason  I wrote two conditions is that these conditionals are very common signals and most commonly used in NUI.